mapbook-iOS/View controllers/Map Packages List/MapPackagesListViewController.swift

```diff
@@ -459,6 +459,7 @@ class LocalPackageCell: UITableViewCell {
     @IBOutlet weak var updateButton:UIButton!
     @IBOutlet weak var activityIndicatorView:UIActivityIndicatorView!
     @IBOutlet weak var updateStackView:UIStackView!
+    @IBOutlet weak var updateStackActivityView: UIStackView!
```

Comment:
My only suggestion would be to change the name of this property so that the word `stack` comes immediately before `view` (since the type is `UIStackView`. So something like `updateActivityStackView` or `updateActionStackView`.